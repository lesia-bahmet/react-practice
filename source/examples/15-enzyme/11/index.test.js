// Core
import { Parent } from './Parent';
import { mount } from 'enzyme';

const result = mount(<Parent />);

describe('example-11', () => {
    test('test-1', () => {
        expect(result).toMatchSnapshot();
    });
});
