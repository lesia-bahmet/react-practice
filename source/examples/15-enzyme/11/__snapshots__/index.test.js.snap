// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example-11 test-1 1`] = `
<bound Parent>
  <section>
    <h1>
      Welcome!
    </h1>
    <p>
      Lorem ipsum dolor sit amet!
    </p>
    <bound Child>
      <span>
        A Child component!
      </span>
    </bound Child>
  </section>
</bound Parent>
`;
